src/lib.rs

```diff
@@ -26,7 +28,7 @@ async fn me() -> String {
     format!("registry_api crate me")
 }
 
-pub async fn spawn_within_actix(config: Config) {
+pub async fn spawn_within_actix<F>(config: Config, config_fn: F) where F: FnOnce(&mut ServiceConfig) + Send + Copy + Clone + 'static {
```

Comment:
Can you rename this function to `spawn_within_actix_custom` and call it through `spawn_within_actix` with empty default `config_fn`? Otherwise we need a minor version bump since this is a breaking change.

Reply:
sure